staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/helpers.go

```diff
@@ -57,3 +59,8 @@ func (s ByPriority) Less(i, j int) bool {
 	}
 	return s[i].Spec.Priority < s[j].Spec.Priority
 }
+
+func APIServiceNameToGroupVersion(apiServiceName string) schema.GroupVersion {
+	tokens := strings.SplitN(apiServiceName, ".", 2)
```

Comment:
> err if len(tokens) != 2

I'll add doc for it to be a valid name.  It's already enforced in API validation.